### ISSUE TITLE
Enhancement: Extended Sequence Simulation

### ISSUE THREAD

**Issue by Schrubbi** [dingram/glypher#7]
Sequence Simulation is a great new feature!
To further improve it would be nice to have presets that behave like specific portal levels, in terms of number of glyphs and time limit (Also a random level option could be cool here).

**Comment by eduard-sukharev** [comment]
Agreed. Some scale to adjust portal level or sequence length would be very nice. Maybe ask the length of upcoming sequence after each try? Or provide two more buttons ("progress" & "degrade") to go to longer/shorter sequence on the sequence result screen?

**Comment by timetracker2643** [comment]
the timing needs to be slowed down and on easy mode should start off with the lowest amount like one or two before moving onto harder ones, after all those that start doing this in the game are more than likely going to try and hit the lower levels to learn and practice from, and that's what you want in the first place.
